sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -662,6 +662,15 @@ case class ShowCurrentNamespace(catalogManager: CatalogManager) extends LeafComm
     AttributeReference("namespace", StringType, nullable = false)())
 }
 
+/**
+ * The logical plan of the SHOW CATALOGS command.
+ */
+case class ShowCatalogs(catalogManager: CatalogManager) extends LeafCommand {
+  override val output: Seq[Attribute] = Seq(
+    AttributeReference("catalog", StringType, nullable = false)(),
+    AttributeReference("default-namespace", StringType, nullable = false)())
```

Comment:
Why we need this？

Reply:
schema of output

Reply:
@yaooqinn Sorry. I misunderstood what you mean. `default-namespace` really doesn't need. I need to maintain consistent behavior with `SHOW TABLES` `SHOW DATABASES` that just show `name`. Thank you very much for reminding@yaooqinn

Reply:
A mistake:  Output of `SHOW TABLES` contains three columns `namespace`,`tableName`,`isTemporary`.  It is different from the rds that just show `name`.